src/utilities/codemirror/vml-adaptation.ts

```diff
@@ -0,0 +1,114 @@
+import { type CompletionContext, type CompletionResult } from '@codemirror/autocomplete';
+import { syntaxTree } from '@codemirror/language';
+import type { ChannelDictionary, CommandDictionary, FswCommand, FswCommandArgument } from '@nasa-jpl/aerie-ampcs';
+import { getNearestAncestorNodeOfType } from '../sequence-editor/tree-utils';
+import { RULE_FUNCTION_NAME, RULE_ISSUE, RULE_STATEMENT, TOKEN_STRING_CONST } from './vml-constants';
+import { getArgumentPosition } from './vml-tree-utils';
+
+export function vmlAutoComplete(
+  _channelDictionary: ChannelDictionary | null,
```

Comment:
Does this function need the `_channelDictionary` argument if it's not being used?

Reply:
addressed in https://github.com/NASA-AMMOS/aerie-ui/pull/1472/commits/04a1ae3e63362cd962e5c192b3acf67d19efbf87